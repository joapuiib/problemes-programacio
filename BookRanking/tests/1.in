3
Head First Java (A Brain Friendly Guide)
Kathy Sierra
059600920-8
78
2020
Java: A Detailed Approach to Practical Coding
Nathan Clark
154512207-5
120
1998
Effective Java
Joshua Bloch
978013468599-1
220
2019
