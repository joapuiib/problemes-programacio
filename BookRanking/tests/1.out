### Llibre amb més pàgines
Effective Java - Joshua Bloch - 2019

### Llibres per any
Head First Java (A Brain Friendly Guide) - Kathy Sierra - 2020
Effective Java - Joshua Bloch - 2019
Java: A Detailed Approach to Practical Coding - Nathan Clark - 1998

### Mitjana de pàgines
139.3

### Llibres publicats després del 2018
Head First Java (A Brain Friendly Guide) - Kathy Sierra - 2020
Effective Java - Joshua Bloch - 2019

### Suma de anys per pàgines
841500

### Més de 100 pàgines
Effective Java
Java: A Detailed Approach to Practical Coding
